### DISCUSSION TITLE
`enableClear` for SearchSelect

### DISCUSSION THREAD

**Discussion by sudhamjayanthi** [tremorlabs/tremor-npm#680] [Ideas]
`<Search />`, `<MultiSelect />` both have prop called `enableClear` which lets the user clear the search input but `<SearchSelect />` doesn't — I'm curious why it wasn't included and if it's possible that this will change?

**Comment by sudhamjayanthi** [comment]
cc: @mitrotasios ?

**Reply by christopherkindl** [reply]
Thanks for spotting! Will be integrated soon and was not neglected on purpose.

**Reply by sudhamjayanthi** [reply]
> Thanks for spotting! Will be integrated soon and was not neglected on purpose.

hey, nvm.

i just fixed it and made a pr (#681) already :D 

check it out and approve if everything looks good!
